code-studio/src/js/code-studio.js

```diff
@@ -16,6 +16,8 @@ require('./videos');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.ColorPicker = require('react-color').default;
```

Comment:
Brad's explanation of why we (unfortunately) have React as a global still is accurate. Using something like browserify-shim seems like the might approach if we can get it working (however it sounds like we havent been able to thus far?). It does seem unfortunate to be sticking this in code-studio, just so that we can stick it on a global to be consumed by apps :(